invenio/base/templates/admin_footer_base.html

```diff
@@ -0,0 +1,57 @@
+{#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+#}
+
+{#
+# This file contains footer part of admin page template.
+#
+# Usage:
+# {% include 'admin_footer.html' %}
+#}
+
+{%- block footer -%}
+<footer class="main-footer">
```

Comment:
Footer is very similar to the existing one - https://github.com/inveniosoftware/invenio/blob/master/invenio/base/templates/footer_base.html
It would be great if they could be harmonised some how so we avoid duplicated code.